### PR TITLE
Webpack 2

### PR DESCRIPTION
**Pull request by nkzawa**
This PR upgrades webpack to v2.

~~btw It looks our blocker (https://github.com/webpack/watchpack/pull/33) is going to be published soon.~~

~~https://github.com/webpack/watchpack/releases/tag/v1.1.2 (not released to npm yet)~~

It seems the tag was removed. I don't know the current status of it.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -77,37 +78,36 @@ gulp.task('build', [
...
+      rules: [
         {
           test: /\.js$/,
           exclude: /node_modules/,
-          loader: 'babel',
-          query: {
-            'babelrc': false,
-            'presets': [
+          loader: 'babel-loader',
+          options: {
```
Comment (rauchg) [inline gulpfile.js]: here we could decorate this with `options.babel()` to make it easier to config babel for next?

**Comment by nkzawa** [comment]
It seems this is a blocker for v2 release. I will add the monkey-patch if needed.

**Comment by arunoda** [comment]
Yep. That's seems like the best way to do it.

**Comment by nkzawa** [comment]
https://github.com/zeit/next.js/pull/449/commits/fce9f02b4e5f54a4f2eb2a0f3575ab8356fa0303 is the monkeypatch

**Pull request merged**